Lakat/i18n/Lakat.alias.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$specialPageAliases = [];
+
+$specialPageAliases['en'] = [
+	'CreateBranch' => ['Create Branch'],
+];
```

Comment:
What is the idea behind this file? It's for language compatibility, right?

Reply:
See comment above for explainations why this is necessary.

Reply:
Thanks